src/components/AlbumForm.vue

```diff
@@ -375,52 +375,44 @@ export default {
         album_artists: [],
       };
 
-      const promises = [];
-
-      for (let label of this.newAlbum.album_labels) {
+      const label_map = this.newAlbum.album_labels.map(async (label) => {
```

Comment:
There is also a small semantic change here (and this happens in almost every form/edit component). Previously, known labels would be added to the transformed object immediately. Now this will only happen when the `await Promise.all` happens below. I don't think this will actually result in a behavioural problem, so I guess it's fine.

Reply:
I did test this, so the behaviour in the end will be the same.
I guess there could be a way to do it immediately, but i fear readability would suffer a lot

Reply:
The way to do it immediately would be the same as it was before: looping over it and adding it to an array in the case of an asynchronous operation.